.github/actions/build_info/action.yml

```diff
@@ -1,35 +1,23 @@
 name: Build info
 description: Sets common builds variables that are needed for other build steps
 
-inputs:
-  force-canary:
-    description: |
-      Forces the current build to be canary.
-      Canary builds test all Python versions and do not use constraints.
-    required: false
-    default: "false"
-
 outputs:
   PYTHON_VERSIONS:
     description: |
       Python versions for which tests will be run.
       The oldest version is replaced by the literal "min" and
       the newest is replaced by the literal "max".
-    value: ${{ steps.build_info.outputs.PYTHON_VERSIONS }}
+    value: '["min", "3.10", "3.11", "3.12", "max"]'
   PYTHON_MIN_VERSION:
     description: Oldest version of Python supported by Streamlit
-    value: ${{ steps.build_info.outputs.PYTHON_MIN_VERSION }}
+    value: "3.9"
   PYTHON_MAX_VERSION:
     description: Latest version of Python supported by Streamlit
-    value: ${{ steps.build_info.outputs.PYTHON_MAX_VERSION }}
+    value: "3.13"
 
```

Comment:
It would be nice at some point that we automatically get min/max from the streamlit setup.py. However, the current version also requires manual configuration, and its not too bad since this is already a centralized place shared across all workflows and changing version support does not happen often.